packages/grafana-prometheus/src/datasource.ts

```diff
@@ -511,144 +502,6 @@ export class PrometheusDatasource
     };
   }
 
-  async annotationQuery(options: AnnotationQueryRequest<PromQuery>): Promise<AnnotationEvent[]> {
-    if (this.access === 'direct') {
```

Comment:
We don't need this anymore right?

Reply:
@zoltanbedi Yes this is the old, deprecated way of handling annotations. So we remove it.